Apps/W1/HybridSL/app/src/codeunits/SLIRSFormData.Codeunit.al

```diff
@@ -0,0 +1,184 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.DataMigration.SL;
+
+using Microsoft.Finance.VAT.Reporting;
+
+codeunit 47009 "SL IRS Form Data"
+{
+    Permissions = tabledata "IRS Reporting Period" = RIM,
+                tabledata "IRS 1099 Form" = RIM,
+                tabledata "IRS 1099 Form Box" = RIM,
+                tabledata "IRS 1099 Form Statement Line" = RIM,
+                tabledata "IRS 1099 Form Instruction" = RIM;
+
+    var
+        IRSFormStatementLineFilterExpressionTxt: Label 'Form Box No.: %1', Comment = '%1 = Form Box No.', Locked = true;
+
+    internal procedure CreateIRSFormsReportingPeriodIfNeeded(ReportingYear: Integer)
+    var
+        IRSReportingPeriod: Record "IRS Reporting Period";
+        PeriodNo: Code[20];
+    begin
+        PeriodNo := Format(ReportingYear);
```

Comment:
We should not duplicate the code. Please use the codeunit 10039 "IRS Forms Data" from the IRS app.
I will remove the access internal form it.